packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -38,6 +38,17 @@
             text-align: center;
         }
     }
+    &__icon {
+        cursor: pointer;
+        transition: transform 0.1s ease-in-out, opacity 0.3s linear;
```

Comment:
@amirghadimi-fs , are the transition effects added confirmed by the design team? 
If possible, please get their approval first before adding the effects if possible. 🙏🏻